index.css

```diff
@@ -0,0 +1,39 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    border: 2px solid red;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    height: 100vh;
+}
+
+#app {
+    border: 2px solid green;
+    height: 600px;
+    width: 900px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+main {
```

Comment:
controlling height is never great - you could use padding on parent component instead

Reply:
In an other way but solved